packages/core/src/errors.ts

```diff
@@ -110,6 +110,7 @@ export const enum RuntimeErrorCode {
   UNSAFE_IFRAME_ATTRS = -910,
   VIEW_ALREADY_DESTROYED = 911,
   COMPONENT_ID_COLLISION = -912,
+  IMAGE_PERFORMANCE_WARNING = 913,
```

Comment:
Since we have a guide, we make this number negative, which indicates to the code that we should produce a link to the error guide page (see a comment about this above in this file). 
```suggestion
  IMAGE_PERFORMANCE_WARNING = -913,
```

Reply:
Oh, I was wondering about the minus signs, and I didn't spot that comment higher up. Good to know!